sql-plugin/pom.xml

```diff
@@ -615,7 +620,7 @@
                                         <!-- inherit from 331+ with exceptions -->
                                         <patternset refid="spark331+.pattern"/>
                                         <exclude name="*until340*/*"/>
-
+                                        <include name="post340/*"/>
```

Comment:
why do we have post340?  Isn't this 341+? this is new naming convention haven't seen before so what are we trying to capture, its also in a spark340.sources which doesn't seem to match

Reply:
similar to how we consume upstream pre320 and post320 TreeNode.scala in Databricks